Source/Upstream: Yes, in v238-stable branch

From 42ae2624942412fff6bc84bfa9d50bb0033e5b9c Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Thu, 8 Mar 2018 22:19:35 +0900
Subject: [PATCH 08/51] sd-bus: do not try to close already closed fd (#8392)

Fixes #8376, which is introduced by 2b33ab0957f453a06b58e4bee482f2c2d4e100c1.

(cherry picked from commit 280029d18f470a64403d68717eef1be5274ff8af)
---
 src/libsystemd/sd-bus/bus-socket.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/libsystemd/sd-bus/bus-socket.c b/src/libsystemd/sd-bus/bus-socket.c
index b5160cff6..166fba157 100644
--- a/src/libsystemd/sd-bus/bus-socket.c
+++ b/src/libsystemd/sd-bus/bus-socket.c
@@ -960,8 +960,6 @@ int bus_socket_exec(sd_bus *b) {
         if (r == 0) {
                 /* Child */
 
-                safe_close(s[0]);
-
                 if (rearrange_stdio(s[1], s[1], STDERR_FILENO) < 0)
                         _exit(EXIT_FAILURE);
 
-- 
2.16.3

